Upload API - final changes (#2957)
wix-incubator/wix-code-docs: wix-media-v2/wix-media-v2/Files/guides/resumable-upload-api.md

}
```

## Status/Error Codes
Errors from this endpoint will include an HTTP status code.
        
### Use uploaded media on your site

> After making the REST HTTP request, you can either display the media on your site directly from the [Media Manager](https://support.wix.com/en/article/wix-media-about-the-media-manager#accessing-the-media-manager-1), or you can set the uploaded media to `$w` frontend elements programatically.

Reviewer: ```suggestion
After making the REST HTTP request, you can either display the media on your site directly from the [Media Manager](https://support.wix.com/en/article/wix-media-about-the-media-manager#accessing-the-media-manager-1), or you can set the uploaded media to `$w` frontend elements.
```

> In order to display the media files on the frontend of your site with code, you first need the file's Media Manager URL.

Reviewer: ```suggestion
To display the media files on the frontend of your site with code, you first need to retrieve the file's Media Manager URL.
```

> To retrieve the Media Manager URL and update your frontend elements:

Reviewer: ```suggestion
To retrieve the Media Manager URL:
```

1.  Call [`getFileDescriptor()`](https://www.wix.com/velo/reference/wix-media-v2/files/getfiledescriptor), passing in `file.id` from the Upload API's [response object](#Response) as its only parameter. 
2. Retrieve the valid Media Manager URL from the `media` property in the `FileDescriptor` object that was returned in the previous step.

> 3. Set the value of the associated `media` property to the [`src`](https://www.wix.com/velo/reference/$w/image/src) property in your site's `$w` frontend elements. 

Reviewer: ```suggestion
3. To use the media in a `$w` element, set the value of the associated `media` property to the element's `src` property. 
```
